simulation/tools/sim_viewer.cpp

```diff
@@ -548,12 +548,13 @@ main(int argc, char** argv)
 
   // Go through VTK files
   for (std::size_t i = 0; i < vtk_file_indices.size(); ++i) {
+    const char* vtk_file = argv[vtk_file_indices[i]];
```

Comment:
Any reason for not using `std::string` here? (You've used one below in similar conditions: mixed use of const char* and std::string)

Reply:
Had it first with `std::string` - but vtk  (`SetFileName`) and `print_value` expecting a `char*`